team_comps.py

```diff
@@ -5,35 +5,47 @@
 #         print("KEY = " + key)
 #     return tier_S
 
-def get_data():
-    pprint(data)
-    return data
 
-def build_data(input):
-    user_data.append(input)
-    print(user_data)
+def get_team_data():
+    pprint(team_data)
+    return team_data
+
+
+def add_champ(champ_name):
+    user_team.append(champ_name)
+    print(user_team)
+
+
+def delete_champ(deletion_input):                            # -champion_name
+    champ_name_length = len(deletion_input) - 1              # length = len(champion_name)
+    user_team.remove(deletion_input[-champ_name_length:])    # remove("champion_name")
+    print(user_team)
+
```

Comment:
Error catching as mentioned above, to handle champions that don't exist in the user_team list.